fl4health/utils/dataset_converter.py

```diff
@@ -0,0 +1,159 @@
+from abc import ABC
+from typing import Callable, Optional, Tuple, Union
+
+import torch
+
+from fl4health.utils.dataset import BaseDataset
+
+
+class DatasetConverter(ABC, BaseDataset):
+    """Dataset converter classes are designed to re-format any dataset for a given training task,
```

Comment:
For our docstrings, we've been putting class descriptions under the __init__ function. So I'd recommend moving this under that method 🙂